src/main.rs

```diff
@@ -244,8 +244,8 @@ fn get_threshold_and_shares() -> (u32, u32) {
     }
 }
 
-fn object_to_file(id: u16, data: &[u8]) -> Result<String, String> {
-    let path_string = format!("./0x{:04x}.yhw", id);
+fn object_to_file(id: u16, object_type: ObjectType, data: &[u8]) -> Result<String, String> {
+    let path_string = format!("./0x{:04x}-{:?}.yhw", id, object_type);
```

Comment:
don't rely on the Debug implementation

Reply:
What do you mean?

Reply:
don't print using `{:?}`, that uses the Debug trait. Print using `{}` which uses the Display trait.

Reply:
I C. Ok